dsgrid/utils/spark.py

```diff
@@ -129,20 +129,26 @@ def _post_process_dataframe(df, cache=False, require_unique=None):
                     raise DSGInvalidField(f"DataFrame has duplicate entries for {column}")
 
 
-def get_unique_values(df, column):
-    """Return the unique values of a dataframe in one column.
+def get_unique_values(df, columns: [str, list]):
```

Comment:
Are you sure this is correct? I think it might be
```
from typing import AnyStr, List, Union
def get_unique_values(df, columns: Union[AnyStr, List]):
```